ksql-common/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchema.java

```diff
@@ -244,16 +243,27 @@ private LogicalSchema rebuild(
         builder.add(Column.of(c.name(), c.type(), Namespace.VALUE, valueIndex++));
       }
 
+      if (windowedKey) {
+        builder.add(
+            Column.of(SchemaUtil.WINDOWSTART_NAME, SqlTypes.BIGINT, Namespace.VALUE, valueIndex++));
+        builder.add(
+            Column.of(SchemaUtil.WINDOWEND_NAME, SqlTypes.BIGINT, Namespace.VALUE, valueIndex++));
+      }
```

Comment:
Bit of a hack - but gets us across the finish line and will be cleaned up with structured key work.